packages/shared/src/utils/performance-logger.ts

```
@@ -0,0 +1,99 @@
+    blockNumber?: number;
+    transactionHash?: string;
+    chainId?: number;
+    roundId?: string;
+    applicationId?: string;
+    donorAddress?: string;
+    recipientAddress?: string;
+    amount?: string;
+    amountInUsd?: string;
+    uniqueDonorsCount?: number;
+    totalDonationsCount?: number;
+    details?: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)